samples/subsys/fs/fs_sample/boards/ek_ra8d1.conf

```diff
@@ -0,0 +1 @@
+CONFIG_DISK_DRIVER_SDMMC=y
```

Comment:
This should be set automatically, no need to do it here

Reply:
Thanks for letting me know. I removed it (for the other tests/samples also)